pages/profile.php

```diff
@@ -207,53 +211,56 @@ function userProfile() {
         echo "
 		<script>
 		var remainingtime = " . (59 - $time_since_last_regen) . ";
-
-		var health = {$player->health};
-		var max_health = {$player->max_health};
-
-		var chakra = {$player->chakra};
-		var max_chakra = {$player->max_chakra};
-
-		var stamina = {$player->stamina};
-		var max_stamina = {$player->max_stamina};
+        var statusBars = {
+            health: [{$player->health}, {$player->max_health}, 0],         
+            chakra: [{$player->chakra}, {$player->max_chakra}, 0],
+            stamina: [{$player->stamina}, {$player->max_stamina}, 0],     
+        };
 
 		var regen = {$player->regen_rate} + {$player->regen_boost}; //no regen cut
 
 		setInterval(() => {
 
-			document.getElementById('regentimer').innerHTML = remainingtime; //minus 1 to compensate for lag
+			$('#regentimer').text(remainingtime); //minus 1 to compensate for lag
```

Comment:
```suggestion
			document.getElementById('regentimer').innerHTML = remainingtime; //minus 1 to compensate for lag
```

Let's keep this as is, I'd like to drop jQuery soon and use native HTML methods